kamailio/db_scripts/db_extra_check.sql

```diff
@@ -0,0 +1,23 @@
+
+do_db_extra_check() {
+
+# location
+KazooDB -db ${DB_CURRENT_DB} "delete from location where socket not like 'udp:%';"
```

Comment:
Is it mean TCP and TLS client not more supported?

Reply:
why not embed this and follow command into kamailio config?
as example define `kamailio:mod-init` (like [htable:mod-init](https://kamailio.org/docs/modules/devel/modules/htable.html#idm1031171732)) and then
```
event_route[kazoo:mod-init] {
    route(STARTUP_CLEAN_LOCATION);
    route(STARTUP_CLEAN_WATCHER);
    ... and etc ...
}
route[STARTUP_CLEAN_LOCATION]
{
    sql_query("exec", 'KZQ_STARTUP_CLEAN_LOCATION_NOT_UDP');
    sql_query("exec", 'KZQ_STARTUP_CLEAN_LOCATION_EXPIRIES');
    sql_query("exec", 'KZQ_STARTUP_CLEAN_LOCATION_ATTRS');
    xlog("L_INFO", "$ci|startup|Cleaned active location table\n");
}
route[STARTUP_CLEAN_WATCHER]
{
    sql_query("exec", 'KZQ_STARTUP_CLEAN_WATCHERS_NOT_UDP');
    sql_query("exec", 'KZQ_STARTUP_CLEAN_WATCHERS_EXPIRIES');
    xlog("L_INFO", "$ci|startup|Cleaned active watchers table\n");
}
```